examples/webgl_loader_xml_vtk.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
```

Comment:
I can try, but the problem is that, I dont have the `.vtp` versions(binary, ascii, and non-compressed) of the bunny model for consistency. If I have that, I can test it out and see if it works. 
Or another way is to create a cube model of the legacy vtk format and then integrate the code... So your pick??